recipes-multimedia/kodi/kodi.inc

```diff
@@ -13,7 +13,7 @@ KODIPATCHES ?= ""
 KODIADDONS ?= ""
 
 SRC_URI = "${KODIURI} ${KODIPATCHES} ${KODIADDONS}"
-SRCREV = "${KODIREV}"
+SRCREV = "6e361cf621a6e90d7f7356e0a053a503f575d06b"
```

Comment:
Some user may use `KODIREV` to alternate version, maybe we should use

KODIREV ??= "${KODIVER}-${KODICODENAME}"
KODIREV ?= "6e361cf621a6e90d7f7356e0a053a503f575d06b"

or just scrap idea behind `KODIREV` entirely

Reply:
First i changed KODIREV instead of SRCREV but reversed that (and kept in mind to change SRCREV to KODIREV again, once we have a 19.2 release). Personally i dont have a need for the additional KODEREV variable but i am pracmatic about that. If you think that KODIREV is in actual use, i'll change it the other way around again.

Reply:
I think we might drop, probably no one using it.